build_js.sh

```diff
@@ -41,19 +43,24 @@ cmake ../src \
     -DCMAKE_PREFIX_PATH="$EMSCRIPTEN" \
     -DCMAKE_TOOLCHAIN_FILE="../src/deps/corrade/toolchains/generic/Emscripten-wasm.cmake" \
     -DCMAKE_INSTALL_PREFIX="." \
-    -DCMAKE_CXX_FLAGS="-s FORCE_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1" \
+    -DCMAKE_CXX_FLAGS="-s FORCE_FILESYSTEM=1 -s ALLOW_MEMORY_GROWTH=1 -s ASSERTIONS=0" \
```

Comment:
@mosra Is this not already covered by the Magnum toolchain? Should it be added for the default Release flags?

Reply:
ASSERTIONS=0 is the default behavior so this change has no effect. I just wanted it to be easier for folks (including myself) to turn on asserts.

This is admittedly unrelated to the rest of the changes in this PR.